fhttp/http_client.go

```diff
@@ -585,11 +588,18 @@ func NewFastClient(o *HTTPOptions) (Fetcher, error) {
 		log.Errf("[%d] Bad url %q : %v", o.ID, urlString, err)
 		return nil, err
 	}
+
+	// Randomly assign a max connection reuse threshold to this thread.
+	var maxConnectionReuse int
+	if len(o.MaxConnectionReuse) == 2 {
+		maxConnectionReuse = o.MaxConnectionReuse[0] + rand.Intn(o.MaxConnectionReuse[1]-o.MaxConnectionReuse[0]+1)
```

Comment:
if max==min maybe not call rand?

and if they are different, shouldn't we reroll each time we close ? (I wonder how we communicate which thread does what / if that's needed)

Reply:
fixed